src/script/backup/BackupRepository.ts

```diff
@@ -308,21 +262,28 @@ export class BackupRepository {
       throw new ImportError(files.error as unknown as string);
     }
 
-    if (!files[Filename.METADATA]) {
-      throw new InvalidMetaDataError();
-    }
-
-    const archiveVersion = await this.verifyMetadata(user, files);
-    const fileDescriptors = Object.entries(files)
-      .filter(([filename]) => filename !== Filename.METADATA)
-      .map(([filename, content]) => {
-        const data = new TextDecoder().decode(content);
-        const entities = JSON.parse(data);
-        return {
-          entities,
-          filename,
-        } as FileDescriptor;
+    // Check for Multiplatform backup
+    if (isMPBackup(files)) {
+      // Import Multiplatform backup
+      const mpbData = await importMPBHistoryToDatabase({
+        backupService: this.backupService,
```

Comment:
MPB/ multiplatform (and now the imports have multiple names too, lets just use one naming for the futures sake)